.github/workflows/main.yml

```
@@ -31,3 +31,7 @@ jobs:
 
       - name: ▶️ Run test script
         run: npm run test
+
+      - name: ⬆️ Upload jest coverage report
+        uses: codecov/codecov-action@v1
```

I think v2 is the recommended version to use now
```suggestion
        uses: codecov/codecov-action@v2
```

---

whoops grabbed from belter. I'll open a PR to belter as well and check for anyone else on v1. good catch!